build/i2c/src/lib.rs

```diff
@@ -719,6 +780,163 @@ impl ConfigGenerator {
         Ok(())
     }
 
+    fn emit_sensor(
+        &mut self,
+        device: &str,
+        label: &str,
+        ids: &Vec<usize>,
```

Comment:
To anticipate clippy: this should be `&[usize]` for maximum generality, callers won't need to change because Vec derefs to a slice.

Reply:
Fixed.